mix.exs

```diff
@@ -39,9 +40,7 @@ defmodule MongoosePush.Mixfile do
       {:plug_cowboy, "~> 2.2"},
       {:jason, "~> 1.4"},
       {:poison, "~> 3.0", override: true},
-      {:distillery, "~> 2.0", override: true},
       {:confex, "~> 3.2", override: true},
-      {:mix_docker, "~> 0.5"},
```

Comment:
Why the `mix_docker` was removed?

Reply:
I removed it because it wasn't used anywhere at all. Additionally, this library is no longer maintained.